spec/motion-spec/matchers/satisfy_spec.rb

```diff
@@ -0,0 +1,18 @@
+# -*- encoding : utf-8 -*-
+describe "Matcher::Satisfy" do
+  it 'satisfy passes when the block returns true' do
+    expect(1).to satisfy{|v| v == 1}
+  end
+
+  it 'satisfy fails when the block returns false' do
+    expect_failure("Block expected to satisfy condition"){ expect(1).to satisfy{|v| v == 3} }
```

Comment:
Line is too long. [93/80]<br>Space missing to the left of {.<br>Space between { and | missing.<br>Space missing inside }.